tools/webidl_binder.py

```diff
@@ -409,6 +423,10 @@ def render_function(class_name, func_name, sigs, return_type, non_pointer,
   call_postfix = ''
   if return_type != 'Void' and not constructor:
     call_prefix = 'return '
+
+  if options.wasm64 and (constructor or return_type in interfaces or return_type == 'String'):
+    call_postfix += ')'
```

Comment:
Maybe move this to new line 441? That would avoid repeating the long condition, and would keep the open and close parens together in this script, showing they work together (as in the if below us, we can have prefix/postfix changes one after the other).

Reply:
Due to the annoying way the prefix and suffix things are constructed here we can't move these together without some refactoring  since  the prefix has to be added before the ones below and the suffix needs to be added after.

I've extract a variable to avoid too much duplication, and perhaps we can clean this up later, but I think that would require a fair bit more refactoring sadly.